### PR TITLE
Adding a new problem

### PR DESCRIPTION
**Pull request by yashikakashyapp**
For assignment 3 of SF-1.

### CODE REVIEW

File: even-in-list/meta.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Even In List",
+  "name": "even_numbers",
```
Comment (domilx) [inline even-in-list/meta.json:3]: Here

File: even-in-list/description.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Write a function called even_list that takes a list of integers as input and returns a list of all of the even integers in the input list. For example: if the input is [1,4,7,9,6,11], your function should return [4,6].
```
Comment (emaj9) [inline even-in-list/description.md:1]: Can you provide 3 example input outputs?

File: even-in-list/io.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "input": [-24, -48, 31, 76, 85, 109],
```
Comment (emaj9) [inline even-in-list/io.json:3]: inputs need double [[ ]]! 

so [[-24, -48.... ]]

File: reverse-string/mutation_1.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+'''
+Simple mistake, this just slices the first and last element of the string instead of changing the order
+'''
+def mutation_1(string):
```
Comment (emaj9) [inline reverse-string/mutation_1.py:4]: Rename the function, should be the same as the name field in meta.json

File: reverse-string/mutation_1.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+'''
```
Comment (emaj9) [inline reverse-string/mutation_1.py:1]: doc strings should be **inside** the function, after the def line

**Comment by emaj9** [comment]
See my comments @yashikakashyapp

**Pull request merged**